### ISSUE TITLE
Suggestions for making IC2 crops less RNG

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/GT-New-Horizons-Modpack#6248]
So if you've done or investigated crops, you know how much of a pain it is to get specific crops, and while adding more is cool, due to the RNG nature, the more you add, the harder it gets to get existing ones. So it would be nice if this were addressed, and people suggested some things to help with this.

So I'll start by giving some suggestions:
1) Have the chance for crops that require a block underneath to mature be significantly increased if that block is already there before you breed a mutation on the farm block above.
2) Require attributes. For example Leavy is on bonsais. So they could ONLY mutate to other bonsais. Obviously we would need to add more attributes to prevent dead-ends and such.
3) Similarly, increase the chance of a mutation to something that has the same attributes by a greater degree. Say 30% for 1, 50% for 2, 60% for 3. This is just an example of course, and is comparing to other random mutations, not including no mutation.
4) Add some kind of mutator spray (similar to the way you use weedex) or something to increase chances or guarantee _some kind_ of mutation
5) Similar to 1, have increased chances if more of that block is within X blocks area. This might be difficult to apply to many crops in the same way, of course, since about half (I think?) don't require anything underneath, nor have a block you can place nearby
6) Similar to 5, some kind of mutator block that requires input (mutagen?)
7) Allow specific mutations in the gendustry stuff
8) Agricraft ripoff
9) Biome-specific chance increase/decrease. Maybe based on bee temp/humidity data
10) Bee interaction of some kind
11) Magic stuff? Craft specific ones, place stuff to increase chances, whatever
12) Different cropsticks that have higher mutation chance, but produce no drops

**Comment by glowredman** [comment]
My thoughts:
1: _Probably_ easy to implement, but would have no effect on crops like Stickreed or Oilberries and making every crop require a certain block would overcomplicate things
2: I think it restricts the player more than it helps narrowing the results down
3: Would reward players, that research the optimal breeding-pair with a higher succes rate, also easy to implement I guess
4/6/12: Some way to increase mutation rate is a viable way in my opinion, but it should not be as lategame as the Gendustry Mutagen
5: Overcomplicated, the issues you already mentioned and might be performance-intensive
7: Maybe can easyly be done, but I think only in combination with any of the other solutions
8: Can possibly buff crops way too much
9: Why not, but sometimes it can be really difficult to find a specific biome, temperature/humididty sounds better to me
10: Sounds a bit like 6. Maybe putting an active Bee House/Apiary near the crops makes them cross-breed faster/have a higher mutation-rate? I don't know how easy it would be to implement nor how impactful it would be.
11: Pretty vague, also sounds like 6 in a way.

In conclusion, I think that a combination of 3 and 9 is something to consider. How much more impactful the attributes/temperature/humidity should be should be tested when such a system is implemented.

**Comment by chochem** [comment]
while I am not against crops being more deterministic, this would result in a massive buff to crops. An crops are imo already pretty good. So I would only be in favor of such a change if there is also a big nerf happening at the same time: (less output, a lot more power needed across the board, specific nerfs to some powerful crops like transformium,...)
